morphia/src/test/java/org/mongodb/morphia/TestQuery.java

```diff
@@ -369,37 +391,67 @@ public void testElemMatchQueryCanBeNegated() {
 
         Mapper mapper = getMorphia().getMapper();
 
-        List<Key<PhotoWithKeywords>> keys = getDs().find(PhotoWithKeywords.class)
+        validate(pwk1, pwk2, pwk3, pwk4, mapper, getDs().find(PhotoWithKeywords.class)
```

Comment:
a bit, yeah.  there's only the one negation at the end, really.  i'll brainstorm some new names.  to be honest, i think this test duplicates a number of others and can probably be combined...